fixtures/fixture.py

```diff
@@ -157,7 +156,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self._cleanups()
         finally:
             self._remove_state()
-        return False # propogate exceptions from the with body.
+        return False  # propogate exceptions from the with body.
```

Comment:
Here is another propogate. I'm sorry I didn't mention it in the previous review.
The rest of the changes look good.